mapdata/WarcraftLegacies/AbilityData/Skin/1093873729.json

```
@@ -72,13 +72,13 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Harvests lumber from trees."
+      "Value": "Harvest Gold from gold mines"
```

Let's keep the original grammar for these:
```suggestion
      "Value": "Harvests gold from gold mines."
```

---

Naturally you'll have to do this change for the other Harvest abilities as well, looks like there are quite a few.